.github/workflows/codecov.yml

```
@@ -11,7 +11,7 @@ jobs:
       - name: Setup python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.10
```

It looks like the `actions/setup-python` GHA expects a string instead of a number, so interprets `3.10` as the number 3.1.
```suggestion
          python-version: '3.10'
```